packages/assets-controllers/src/NftController.ts

```diff
@@ -409,24 +404,24 @@ export class NftController extends BaseController<
    * Handles the state change of the preference controller.
    * @param preferencesState - The new state of the preference controller.
    * @param preferencesState.ipfsGateway - The configured IPFS gateway.
-   * @param preferencesState.openSeaEnabled - Controls whether the OpenSea API is used.
+   * @param preferencesState.displayNftMedia - Controls whether the NFT API is used.
    * @param preferencesState.isIpfsGatewayEnabled - Controls whether IPFS is enabled or not.
    */
   async #onPreferencesControllerStateChange({
     ipfsGateway,
-    openSeaEnabled,
+    displayNftMedia,
```

Comment:
I'm wondering if this is going to work on both mobile and extension.  I think the core preferences controller is only used by mobile.  And extension has its own version.  But then how will extension work since it still has `openSeaEnabled`?

Reply:
Hmmmm, Was thinking to rename the variable on extension also since technically we no longer use opensea (might require us to do migrations to update user state) ;

On extension; when a user triggers and update of the openseaEnabled toggle for exp; controllerMessenger will publish the event `"PreferencesController:stateChange"` [here](https://github.com/MetaMask/metamask-extension/blob/8f2bab54f6cb8bc092348b0863bdb4239398efc0/app/scripts/metamask-controller.js#L7082) which controllers like NftController subscribe to and update their state accordingly;

When controllerMessenger on extension publishes the event its also sending the currentState which is an object containing "openseaEnabled"
```
this.controllerMessenger.publish(
      'PreferencesController:stateChange',
      currentState,
      [],
);
```
Then controllers which subscribe to `PreferencesController:stateChange` would read the value
```
async #onPreferencesControllerStateChange({
    ipfsGateway,
    displayNftMedia,
    isIpfsGatewayEnabled,
  }: PreferencesState) {
.....
  }
```
A quicker way would be to do this instead in core:
```
async #onPreferencesControllerStateChange({
    ipfsGateway,
    openseaEnabled: displayNftMedia,
    isIpfsGatewayEnabled,
  }: PreferencesState) {
.....
  }
```

Reply:
I do think we'll want to migrate extension from `openSeaEnabled` to `displayNftMedia` so that it's aligned with mobile.  Doesn't have to block this PR.  Just would have to do it before extension can consume this.